pkgs/development/python-modules/dipy/default.nix

```
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, packaging
+, pytest
+, python
+, cython
+, numpy
+, scipy
+, pandas
+, scikitlearn
+, statsmodels
+, h5py
+, tables
+, cvxpy
+, matplotlib
+, nibabel
+, withCvxpy ? false  # unfree dependency on `mkl`
+}:
+
+buildPythonPackage rec {
+  pname = "dipy";
+  version = "1.1.1";
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner  = "nipy";
```

redirects to dipy for me
```suggestion
    owner  = "dipy";
```

---

Changed.